NitroxClient/MonoBehaviours/Gui/Settings/SettingsManager.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Steamworks;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours.Gui.Settings
+{
+    public static class SettingsManager
+    {
+        public static string Name { private get; set; }
```

Comment:
I can't set the default value with ```SteamFriends.GetPersonaName()``` because I have to check if SteamAPI is there (```SteamAPI.Init()```). I need a function and to be save nobody get a null value because the get is public I set it private.

Reply:
You can call that in the static constructor, or the property get method. Up to you to see what looks best.